### PR TITLE
Release  2023.1

### PR DESCRIPTION
**Pull request by cgwalters**
Release 2023.1

---

configure: post-release version bump

---

### CODE REVIEW

**Comment by jlebon** [comment]
```
Verifying all public symbols are documented:
+ cat
+ sha256sum -c released-sha256.txt
sha256sum: WARNING: 1 computed checksum did NOT match
ok documented symbols
PASS: tests/test-symbols.sh 2 documented symbols
++ report_err
++ local exit_status=1
Unexpected nonzero exit status 1 while running: sha256sum -c released-sha256.txt
```

Missing a checksum update?

File: configure.ac [hunk 1]
```diff
@@ -1,7 +1,7 @@
 AC_PREREQ([2.63])
 dnl To perform a release, follow the instructions in `docs/CONTRIBUTING.md`.
-m4_define([year_version], [2022])
-m4_define([release_version], [8])
+m4_define([year_version], [2023])
+m4_define([release_version], [2])
```
[the remark this answers is not in the document]
Comment (dbnicholson) [reply configure.ac:4]: Nevermind, I didn't look at the individual commits.

[the remark this answers is not in the document]
Comment (cgwalters) [reply configure.ac:4]: It is in the first commit; the second is the post-release version bump.

File: Makefile-libostree.am [hunk 2]
```diff
@@ -173,9 +173,9 @@ endif # USE_GPGME
 symbol_files = $(top_srcdir)/src/libostree/libostree-released.sym
 
 # Uncomment this include when adding new development symbols.
-if BUILDOPT_IS_DEVEL_BUILD
-symbol_files += $(top_srcdir)/src/libostree/libostree-devel.sym
-endif
+#if BUILDOPT_IS_DEVEL_BUILD
+#symbol_files += $(top_srcdir)/src/libostree/libostree-devel.sym
+#endif
```
[the remark this answers is not in the document]
Comment (cgwalters) [reply Makefile-libostree.am:178]: It's because the linker complains about an empty symbol file.

**Review by jlebon — APPROVED** [review]

File: Makefile-libostree.am (hunk 2 above)
[the remark this answers is not in the document]
Comment (dbnicholson) [reply Makefile-libostree.am:178]: Ah.

**Pull request merged**